.github/workflows/deploy-to-production.yml

```
@@ -35,27 +41,80 @@
-            EOF
+    name: Deploy to production server 🚀
+    runs-on: ubuntu-latest
+    needs: merge-images-into-manifest-list
+    steps:
+      - name: Deploy to production server 🚀
+        env:
+          SSH_PRIVATE_KEY: ${{ secrets.PRODUCTION_SERVER_SSH_PRIVATE_KEY }}
+          SSH_USER: ${{ secrets.PRODUCTION_SERVER_SSH_USER }}
+          SSH_SERVER_IP: ${{ secrets.PRODUCTION_SERVER_SSH_ADDRESS }}
+          UPDATE_SCRIPT_FULL_PATH: ${{ secrets.PRODUCTION_UPDATE_SCRIPT_FULL_PATH }}
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:6: Quote this to prevent word splitting

(shellcheck)